ibm/service/cdtoolchain/resource_ibm_cd_toolchain_tool_appconfig.go

```diff
@@ -38,38 +38,38 @@ func ResourceIBMCdToolchainToolAppconfig() *schema.Resource {
 				MinItems:    1,
 				MaxItems:    1,
 				Required:    true,
-				Description: "Unique key-value pairs representing parameters to be used to create the tool.",
+				Description: "Unique key-value pairs representing parameters to be used to create the tool. A list of parameters for each tool integration can be found in the <a href=\"https://cloud.ibm.com/docs/ContinuousDelivery?topic=ContinuousDelivery-integrations\">Configuring tool integrations page</a>.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": &schema.Schema{
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: "Type a name for this tool integration, for example: my-appconfig. This name displays on your toolchain.",
+							Description: "The name used to identify this tool integration. App Configuration references include this name to identify the App Configuration instance where the configuration values reside. All App Configuration tools integrated into a toolchain should have a unique name to allow resolution to function properly.",
 						},
-						"region": &schema.Schema{
+						"location": &schema.Schema{
```

Comment:
We shouldn't change or rename the argument names. This fails the backward compactability. We need to follow the guidelines of Hashicorp for these kind of changes